maps/biter_battles_v2/item_costs.lua

```diff
@@ -33,6 +33,42 @@ ItemCosts.raw_costs = {
     -- So that is 150 oil/sec == 30 ore/sec.
     ['crude-oil'] = { cost = 2, crafting_time = 0.1 / 0.8, icon = '[fluid=crude-oil]' },
     ['water'] = { cost = 0, crafting_time = 1 / 1200, icon = '[fluid=water]' },
+
+    --space
+    ['ice'] = { cost = 1, crafting_time = 2, icon = '[item=ice]' },
+
+    --gleba
+    ['jelly'] = { cost = 7, crafting_time = 1, icon = '[item=jelly]' },
+    ['yumako-mash'] = { cost = 14, crafting_time = 1, icon = '[item=yumako-mash]' },
+
+    --vulcanus
+    ['tungsten-ore'] = { cost = 20, crafting_time = 5, icon = '[item=tungsten-ore]' },
+    ['calcite'] = { cost = 1, crafting_time = 2, icon = '[item=calcite]' },
+
+    --fulgora
+    ['holmium-ore'] = { cost = 15, crafting_time = 1 / 1200, icon = '[item=holmium-ore]' },
+
+    --aquilo
+    ['cryogenic-science-pack'] = { cost = 500, crafting_time = 30, icon = '[item=cryogenic-science-pack]' },
+
+    --promethium
+    ['quantum-processor'] = { cost = 200, crafting_time = 30, icon = '[item=quantum-processor]' },
+    ['promethium-asteroid-chunk'] = { cost = 800, crafting_time = 200, icon = '[item=promethium-asteroid-chunk]' },
+
+    ['fluoroketone-hot'] = { cost = 0, crafting_time = 1 / 1200, icon = '[fluid=water]' },
+    ['fluoroketone-cold'] = { cost = 0, crafting_time = 2, icon = '[fluid=water]' },
+    ['lithium-brine'] = { cost = 0, crafting_time = 1 / 1200, icon = '[fluid=water]' },
```

Comment:
Same crafting time as crude oil `crafting_time = 0.1 / 0.8`?
I guess these are placeholders mostly tho.
Should fluorine be here too? It's also from pumpjack on Aquilo.

Reply:
for some reason opening the food values tab only complained about these values missing, so i really just added them.. i dont think they're actually shown anywhere though so these are pure dummy values

Reply:
Oh right these are the things printed when opening food values tab.

It's fine as this. I'm just in reviewer mode lol.